CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 5.0.9
+
+- Make Table config print optional
```

Comment:
maybe include the env var name here as well so it's more discoverable

Reply:
Addressed all:
- Renamed the env to `FSD_UTILS_ENABLE_CONFIG_TABLE`
- Changed the version to 5.1.0, representing a new feature rather than a bug fix
- Updated CHANGELOG